crates/openvino/Cargo.toml

```diff
@@ -16,6 +16,7 @@ exclude = ["/tests"]
 openvino-sys = { path = "../openvino-sys", version = "0.6.0" }
 openvino-finder = { path = "../openvino-finder", version = "0.6.0" }
 thiserror = "1.0"
+anyhow = "1.0"
```

Comment:
I'm not sure I understand why we would need to import this: just using the regular `Result` with the appropriate error type from this crate should be good enough.

Reply:
Oh, I see, you're just using `anyhow` for easier testing... in that case I'll just move it to `[dev-dependencies]` so that it is available during testing but not required by users of the crate.